### PR TITLE
Display corresponding 2fa channel in text according to config

### PR DESCRIPTION
**Pull request by tung2744**
ref DEV-2146
ref DEV-2250

### CODE REVIEW

File: resources/authgear/templates/pl/translation.json [hunk 1]
```diff
@@ -1252,7 +1252,7 @@
   "v2.page.settings-oob-otp-sms.default.title": "{PhoneOTPMode, select, whatsapp {WhatsApp} whatsapp_sms {WhatsApp/SMS} other {SMS} } wiadomość",
   "v2.page.settings-oob-otp.default.added-at-label": "Dodano o <span data-date=\"{rfc3339}\" data-date-type=absolute data-date-date-style=long>{time, datetime, short} UTC</span>",
   "v2.page.settings-oob-otp.default.oob-otp-email-button-label": "+ Dodaj email",
-  "v2.page.settings-oob-otp.default.oob-otp-sms-button-label": "+ Dodaj numer telefonu {PhoneOTPMode, select, whatsapp {WhatsApp} whatsapp_sms {WhatsApp/SMS} other {SMS} } Passkey",
```
Comment (tung2744) [inline resources/authgear/templates/pl/translation.json]: I guess this Passkey is incorrect so I removed it, although I don't know this language

**Comment by louischan-oursky** [comment]
build failed.

**Review by louischan-oursky — APPROVED** [review]

File: resources/authgear/templates/en/translation.json [hunk 2]
```diff
@@ -1236,7 +1236,7 @@
   "v2.page.settings-mfa.default.revoke-device-dialog-description": "You have {NumberOfDeviceTokens, plural, one{# trusted device} other{# trusted devices}} trusted devices associated with your account. Do you want to remove all trusted devices?",
   "v2.page.settings-mfa.default.revoke-device-dialog-title": "Remove trusted devices",
   "v2.page.settings-mfa.default.secondary-oob-otp-email-label": "Email",
-  "v2.page.settings-mfa.default.secondary-oob-otp-sms-label": "WhatsApp/SMS message",
+  "v2.page.settings-mfa.default.secondary-oob-otp-sms-label": "{PhoneOTPMode, select, whatsapp {WhatsApp} whatsapp_sms {WhatsApp/SMS} other {SMS} } message",
```
Comment (louischan-oursky) [inline resources/authgear/templates/en/translation.json:1239]: I have another thought. I think we should split this into three keys

```
v2.page.settings-mfa.default.secondary-oob-otp-sms-label--whatsapp
v2.page.settings-mfa.default.secondary-oob-otp-sms-label--sms
v2.page.settings-mfa.default.secondary-oob-otp-sms-label--whatsapp-or-sms
```

**Review by louischan-oursky — CHANGES_REQUESTED** [review]

**Comment by tung2744** [comment]
Updated to separate into 3 translations

**Comment by tung2744** [comment]
Also handled dev-2250

**Review by louischan-oursky — APPROVED** [review]

**Pull request merged**